resources/views/pages/prompt-tos.blade.php

```diff
@@ -0,0 +1,58 @@
+@extends("templates._page")
+
+@push('head')
+	<style>
+
+		#content {
+			height: 100vh;
+		}
+
+		@media screen and (max-width: 768px) {
+			.btn {
+				width: 100%;
+			}
+		}
+
+		.scroll_box{
+			overflow-y: scroll;
+			height:50%;
+		}
+
+	</style>
+
+	<script>
+
+		function agreement_form(){
+			return {
+				agreed: false,
+				dismissed: false
+			}
+		}
+
+	</script>
+@endpush
+
+@section("content")
+
+
+	<div class="container p-4 h-100" x-data="agreement_form()">
+
+		<div class="alert alert-info" x-show="!dismissed"><a href="#" class="alert-link" style="float: right;" @click="dismissed = true"><i class="fa fa-times"></i></a> Sorry - a lot has changed in the <a class="alert-link" href="{{ route('whats-new') }}">2.0 update</a>, and you'll need to accept the terms of service before you can continue using the site.</div>
+
+		<h1>{{ $title }}</h1>
+		<p><i>{{ $date }}</i></p>
+		<div class="scroll_box mb-4">{!! Markdown::convertToHtml($markdown); !!}</div>
```

Comment:
Give it a bit of separation and some room to breathe: `class="scroll_box mb-4 border p-3 rounded"` maybe?